controllers/authors.js

```diff
@@ -0,0 +1,34 @@
+const models = require('../models')
+
+const getAllAuthors = async (request, response) =>
+{
```

Comment:
Watch your inconsistent spacing. Some places you have the opening brace on the previous line and some places on its own line. Pick one and be consistent.